packages/util-fns/obj_util/index.js

```diff
@@ -0,0 +1,10 @@
+function mapObj(obj, keyTransformFn = x => x, valTransformFn = x => x) {
```

Comment:
It's a good idea to use objects as parameters here. So like this - 
```suggestion
const id = x => x
function mapObj({ obj = {}, keyTransformer = id, valTransformer = id }) {
...
}
```
and then use it like this - 
```js
mapObj({ obj: {}, keyTransformer: myFunc, valTransformer: secondFunc })
```

This way it is clear that which function is transforming keys and which one is transforming the values. Otherwise, that becomes an implicit knowledge or multiple lookups.

What do you say?

Reply:
Don't you think this function is simple enough for people to remember its signature without having to look it up again? Plus, requiring an obj argument might cause multiple lookups if people forget the keys

The current signature is as similar to the widely used Array.prototype.map (as similar as possible, without modifying Object.prototype)